.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,63 @@
+name: Cloud Function CI/CD
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # 0740 UTC every day
+    - cron: '40 7 * * *' 
```

Comment:
Wan't aware that Github Actions had support for cron jobs as well. Nice 💯

Reply:
The other option is cloud scheduler + cloud (run) function, but just seems simpler to go down this road for now without all that extra ceremony.

Reply:
Yep, agree 👍